src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -50,6 +50,9 @@ def get_client(self):
             app_id=self.model.metadata['installation']['id'],
         )
 
+    def get_repositories(self):
+        return self.get_client().get_repositories()
```

Comment:
Should this be here?

Reply:
Well it does for the repositories dropdown to work

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_integration_repos.py#L29

Reply:
Could you change the wording on this commit when you merge in that case to reflect that this actually is for making GHE work with repo stuff?